device/arista/x86_64-arista_7060x6_64pe/Arista-7060X6-64PE-P32O64/buffers_defaults_t1.j2

```diff
@@ -0,0 +1 @@
+BALANCED/buffers_defaults_t1.j2
```

Comment:
Can you add a new symbol link `buffers_default_lt2.j2` which points to `BALANCED/buffers_defaults_t1.j2` at this time being?

Reply:
We can work on another PR to update the symbol link once the buffer configuration is finalized.
@kperumalbfn for viz